kafka-ui-react-app/src/components/Topics/Topic/Messages/MessageContent/__tests__/MessageContent.spec.tsx

```diff
@@ -82,41 +82,41 @@ describe('MessageContent screen', () => {
   });
 });
 
-describe('checking content type depend on message type', () => {
-  it('renders component with message having JSON type', () => {
-    render(
-      setupWrapper({
-        messageContentFormat: 'JSON',
-        messageContent: '{"data": "test"}',
-      })
-    );
-    expect(screen.getAllByText('JSON')[1]).toBeInTheDocument();
-  });
-  it('renders component with message having AVRO type', () => {
-    render(
-      setupWrapper({
-        messageContentFormat: 'AVRO',
-        messageContent: '{"data": "test"}',
-      })
-    );
-    expect(screen.getByText('AVRO')).toBeInTheDocument();
-  });
-  it('renders component with message having PROTOBUF type', () => {
-    render(
-      setupWrapper({
-        messageContentFormat: 'PROTOBUF',
-        messageContent: proto,
-      })
-    );
-    expect(screen.getByText('PROTOBUF')).toBeInTheDocument();
-  });
-  it('renders component with message having no type which is equal to having PROTOBUF type', () => {
-    render(
-      setupWrapper({
-        messageContentFormat: 'PROTOBUF',
-        messageContent: '',
-      })
-    );
-    expect(screen.getByText('PROTOBUF')).toBeInTheDocument();
-  });
-});
+// describe('checking content type depend on message type', () => {
```

Comment:
shouldn't we test it by setting search params in the testing to these values ? then render the component for testing. that way we can test `messageContentFormat`

Reply:
@Mgo i wrote above that i will change test cases if the solution is normal, i know about linter error i commented  some code in test cases